### ISSUE TITLE
Question about _roku.jpg images. What would cause a very small number of them to fail to generate?

### ISSUE THREAD

**Issue by walt93** [WWBN/AVideo#9693]
**Describe the question**
I'm failing to retrieve two images, using the pattern <filename>_roku.jpg.

If I remove _roku from the filename, the image comes up. 

This only happens for 2 out of ≈7,000 videos.

https://conspyre.tv/videos/video_220519001648_v8a00/video_220519001648_v8a00_roku.jpg
No image

https://conspyre.tv/videos/video_220519001648_v8a00/video_220519001648_v8a00.jpg
Image

**Comment by DanielnetoDotCom** [comment]
https://conspyre.tv/videos/video_220519001648_v8a00/video_220519001648_v8a00_roku.jpg is there for me

but roku images are generated when you access the roku feed

**Comment by eitanwa** [comment]
I'm glad the issue was raised
I just wanted to write about it
The feeds none of them (rss roku mrss) extract image nor video only text 
https://alive528.com/feed/?catName=science-and-technology

thanks
@DanielnetoDotCom

**Comment by DanielnetoDotCom** [comment]
@eitanwa your roky feed is this one https://alive528.com/roku.json

and I can see images in it.

**Comment by eitanwa** [comment]
> @eitanwa your roky feed is this one https://alive528.com/roku.json
> 
> and I can see images in it.

can you see the image of the rss feed?
I meant thumbnails
this is from a feed reader (screen shot)

![image](https://github.com/user-attachments/assets/de957540-314f-4bb8-9415-2a8037574eb1)

like this 

![image](https://github.com/user-attachments/assets/4e69e61a-470c-4f97-821f-9387001f7c5e)

**Comment by walt93** [comment]
Yes, @eitanwa I can see thumbnails. Open the URL and you see a big messy JSON structure. 

Select All
Copy
Go to jsonlist.com
Paste
Press "Validate JSON"

Now you can read the mess a bit easier. Find the "thumbnail" URLs and copy/paste into your browser, and they work.

![Screenshot 2025-01-09 at 4 39 10 PM](https://github.com/user-attachments/assets/cd327fe3-8ce5-49d5-aaf4-0cacf9c4c919)

![Screenshot 2025-01-09 at 4 39 28 PM](https://github.com/user-attachments/assets/c551748c-cf09-4ba3-b72f-9204f8d00d43)

**Comment by DanielnetoDotCom** [comment]
I am not aware about any mess
